Cargo.toml

```diff
@@ -13,27 +13,27 @@ default = []
 sdl = ["gfx_window_sdl"]
 vulkan = ["gfx_device_vulkan", "gfx_device_vulkanll", "gfx_window_vulkan"]
 metal = ["gfx_device_metal", "gfx_window_metal"]
+serialize = ["gfx_core/serialize"]
 unstable = []
 
-
 [lib]
 name = "gfx_app"
 
 [dependencies]
 log = "0.3"
-env_logger = "0.3"
-glutin = "0.7.1"
-winit = "0.5.1"
-gfx_core = { path = "src/core", version = "0.6" }
+env_logger = "0.4"
+glutin = "0.7"
```

Comment:
According to semantic versioning, the patch number is for backwards-compatible bug fixes only. By removing the patch version from the `Cargo.toml`, we can get these bug fixes whenever they come out, so we don't have to manually increment the version number over and over again.